packages/@aws-cdk-testing/framework-integ/test/aws-rds/test/integ.cluster-multiaz.ts

```diff
@@ -0,0 +1,39 @@
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import * as kms from 'aws-cdk-lib/aws-kms';
+import * as cdk from 'aws-cdk-lib';
+import { Credentials, MultiAZDatabaseCluster, DatabaseClusterEngine, ParameterGroup } from 'aws-cdk-lib/aws-rds';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-cdk-rds-multiaz-integ');
+
+const vpc = new ec2.Vpc(stack, 'VPC', { maxAzs: 3, restrictDefaultSecurityGroup: false });
+
+const params = new ParameterGroup(stack, 'Params', {
+  engine: DatabaseClusterEngine.MYSQL,
+  description: 'A nice parameter group',
+  parameters: {
+    character_set_database: 'utf8mb4',
+  },
+});
+
+const kmsKey = new kms.Key(stack, 'DbSecurity');
+
+const cluster = new MultiAZDatabaseCluster(stack, 'Database', {
+  engine: DatabaseClusterEngine.MYSQL,
+  credentials: Credentials.fromUsername('admin', { password: cdk.SecretValue.unsafePlainText('7959866cacc02c2d243ecfe177464fe6') }),
+  instanceProps: {
+    instanceType: ec2.InstanceType.of(ec2.InstanceClass.M5D, ec2.InstanceSize.LARGE),
+    vpcSubnets: { subnetType: ec2.SubnetType.PUBLIC },
+    vpc,
+  },
+  parameterGroup: params,
+  storageEncryptionKey: kmsKey,
+});
+
+cluster.connections.allowDefaultPortFromAnyIpv4('Open to the world');
+
+new IntegTest(app, 'cluster-multiaz-test', {
+  testCases: [stack],
```

Comment:
You can add `enableLookups: true` here to be able to create the Vpc with 3 subnets. By default stack lookups are disabled, but for certain tests that require  lookups you can enable it.

Reply:
Finally got back to being able to rebase my branch and try using `enableLookups: true` but am having the same problem where the VPC that gets created only has 2 subnets.  Any other ideas?